### PR TITLE
Update Dependabot ignore directives

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request adds an attribution comment to the Dependabot ignore directives and adds the remaining versioned Actions that should be managed by this project to the list of commented out directives.

## 💭 Motivation and context ##

All versioned Actions that appear in the workflow(s) managed by this repository should be managed here (i.e. downstream projects should uncomment the ignore directives so Dependabot does not attempt to update downstream repositories). Since they are managed here we should also add attribution so it is clear in downstream projects where specific dependencies are being managed.

## 🧪 Testing ##

Automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -12,9 +12,13 @@ updates:
     schedule:
       interval: "weekly"
     # ignore:
+    #   # Managed by cisagov/skeleton-generic
```
Comment (dav3r) [inline .github/dependabot.yml:15]: Good idea! 👍

**Review by dav3r — APPROVED** [review]
👍

File: .github/dependabot.yml (hunk 1 above)
Comment (jsf9k) [reply .github/dependabot.yml:15]: I love it!

**Review by jsf9k — APPROVED** [review]
Great idea!

**Pull request merged**